pages/Guides/supernode-program/docs.en.md

```diff
@@ -6,92 +6,122 @@ taxonomy:
 
-  A tutorial can be found [here](http://nem.ghost.io/how-to-use-delegated-harvesting/).
+## Steps to Enroll and Participate in the Supernode Program
+
+- **Step 1**: Download and Launch Latest NEM NanoWallet From [the GitHub Releases Page](https://github.com/NemProject/NanoWallet/releases).
+
+- **Step 2**: Make an Account and Have at Least 10,010 XEM.
+
+  10,000 XEM to be able to participate in the Supernode Program, and ~10 XEM to pay the fees for activating delegated harvesting and sending the enrollment message. (You will need to re-enroll in the Supernode Program every month - 10 XEM should be enough to cover transaction fees for at least a year.)
+
+- **Step 3**: Activate Delegated Harvesting on Your Main Account.
+
+  The tutorial for NEM NanoWallet delegated harvesting can be found [here](https://nemproject.github.io/nem-docs/pages/Guides/nanowallet/delegated-harvesting/docs.en.html#activation).
+
+  Six (6) hours must pass before you can start harvesting, but enrollment in the Supernode Program can be initiated once you have the delegated private key (available immediately). (Please make sure to get the private key from the **delegated** harvesting account, not your main account and not either of the public keys.)
   
-  Remember, 6 hours must pass before you can start harvesting, but for the sake of supernodes, once you have the delegated private key (which is available instantly), you may proceed. (Please make sure to get the private key from the **delegated** harvesting account, not your main account and not either of the public keys.)
+  It's important to understand the difference between Main and Delegated Key Pairs which are used in NEM:
+  - Main - Is used to control funds for your account. If the main private key will be stolen all funds could be transferred to another account. Never expose or share your Main Private Key!
+  - Delegated - Is used to control delegated harvesting. If the delegated private key will be stolen your funds are safe. You'll be also able to change the compromised delegated account linked to your Main account.
+  
+  For NEM node and Servant configuration you should use **Delegated** keys.
+  To find your delegated private key go to Services -> Delegated Harvesting -> Manage delegated account. Choose "Show delegated account keys", and put the password to reveal the delegated private key.
+
+- **Step 4**: Setup NEM Node.
 
-- **Step 4**: Shut down NIS, if running.
+  If you don't have a NEM node, you need to set up it first. The tutorial can be found [here](https://nemproject.github.io/nem-docs/pages/Guides/node-operation/docs.en.html).
 
-- **Step 5**: Configure your NIS to autoboot with the **delegated** private key.
+- **Step 5**: Shutdown the NIS Client, if Running.
 
-  To find your delegated private key please use NCC and go to *View Account Details* and then *Show Remote Account's PRIVATE Key*.
+- **Step 6**: Configure Your NEM Node to Autoboot With the **Delegated** Private Key.
```

Comment:
If I'm following correctly, this refers to the Delegated private key mentioned in Step 3.. If so, I feel it would be useful to reference Step 3.

i.e.:
`- **Step 6**: Configure Your NEM Node to Autoboot With the **Delegated** Private Key (see Step 3 above)
`
OR
`  You will need to edit the config file in your NIS folder to include the **Delegated** Private Key collected in Step 3. Here is some help:`

Reply:
Changed (option 2)